procedures/foreman/release.md.erb

```diff
@@ -64,22 +64,38 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 - Update [foreman-packaging](https://github.com/theforeman/foreman-packaging) branches
   - [ ] rpm/<%= short_version %> using <%= rel_eng_script('bump_rpm_packaging') %>
   - [ ] deb/<%= short_version %> using <%= rel_eng_script('bump_deb_packaging') %>
+<% if Gem::Version.new(foreman_version) < Gem::Version.new('3.9') -%>
 - [ ] Wait for packages to be built using <%= rel_eng_script('wait_packaging') %> (checks [rpm/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-rpm-<%= short_version %>-release/) and [deb/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-deb-<%= short_version %>-release/)). This takes a while so you can already start on the manual updates.
+<% else -%>
+- [ ] Wait for packages to be built using <%= rel_eng_script('wait_copr_packaging') %> (checks [rpm/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-rpm-copr-<%= short_version %>-release/) and [deb/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-deb-<%= short_version %>-release/)). This takes a while so you can already start on the manual updates.
+<% end -%>
 - Check for outstanding PRs against <%= short_version %> packaging, and merge if possible:
   - [ ] [rpm/<%= short_version%>](https://github.com/theforeman/foreman-packaging/pulls?q=is%3Apr+is%3Aopen+base%3Arpm%2F<%= short_version %>)
   - [ ] [deb/<%= short_version%>](https://github.com/theforeman/foreman-packaging/pulls?q=is%3Apr+is%3Aopen+base%3Adeb%2F<%= short_version %>)
 - Sign the RPMs in the release
+<% if Gem::Version.new(foreman_version) < Gem::Version.new('3.9') -%>
   - [ ] <%= rel_eng_script('download_rpms') %>
   - [ ] <%= rel_eng_script('sign_rpms') %>
   - [ ] <%= rel_eng_script('upload_rpm_signatures') %>
   - [ ] <%= rel_eng_script('upload_rpms') %>
+<% else -%>
+  - [ ] <%= rel_eng_script('generate_stage_repository') %>
+  - [ ] <%= rel_eng_script('sign_stage_rpms') %>
+  - [ ] <%= rel_eng_script('upload_stage_rpms') %>
+<% end -%>
 - Sign RPMs for client repos (call scripts with `PROJECT=client`)
```

Comment:
I am missing plugins here.
Do we intend to keep them unsigned (I'd prefer if we have them signed, TBH) and if we do what other workflow does copr(→stagingyum)→yum for them? Will it be `https://ci.theforeman.org/job/foreman-plugins-${VERSION}-rpm-pipeline/`?

Reply:
Since plugins are not signed (can revisit that separately), the plugin release job can be normal and mimic nightly (straight up generate and push).

Reply:
This is reflected a bit with an update here (https://github.com/theforeman/jenkins-jobs/pull/379) as it keeps plugins chugging along always.